src/ui/actions/timeline.ts

```diff
@@ -256,34 +237,44 @@ export function seek(
     } else {
       ThreadFront.timeWarp(point, time, hasFrames);
     }
+    dispatch(setTimelineToTime(time));
```

Comment:
I don't think this is a good idea: `setTimelineToTime()` paints the graphics but `ThreadFront.timeWarp()` will emit a `paused` event and that will also trigger painting the graphics: 
https://github.com/RecordReplay/devtools/blob/6f3198df7f6720231e6c1a5a71265e0a7b1854f0/src/protocol/graphics.ts#L264-L288
Not only are we painting twice, in the worst case this may actually paint a recorded screenshot over a repainted one, which would be a bug.

Reply:
Oh! Interesting! This is also what I expected, and then I was watching the protocol monitor and it did *not* issue a request for a repaint (and the graphics did not update) so I added this 😢